package.json

```diff
@@ -23,11 +22,10 @@
   },
   "scripts": {
     "setup": "node ./hosting/scripts/setup.js && yarn && yarn bootstrap && yarn build && yarn dev",
-    "bootstrap": "lerna bootstrap",
-    "postbootstrap": "lerna link && ./scripts/link-dependencies.sh",
+    "bootstrap": "./scripts/link-dependencies.sh",
```

Comment:
This is still needed to link the pro and account-portal packages. It will not be needed when everything is a proper submodule and managed via yarn workspace